src/Service/ActivityPub/ApHttpClient.php

```diff
@@ -379,20 +379,28 @@ public function post(string $url, User|Magazine $actor, ?array $body = null): vo
             return;
         }
 
+        $jsonBody = json_encode($body ?? []);
+
         $this->logger->debug("ApHttpClient:post:url: $url");
-        $this->logger->debug('ApHttpClient:post:body '.json_encode($body ?? []));
+        $this->logger->debug("ApHttpClient:post:body $jsonBody");
 
         // Set-up request
-        $client = new CurlHttpClient();
-        $response = $client->request('POST', $url, [
-            'max_duration' => self::MAX_DURATION,
-            'timeout' => self::TIMEOUT,
-            'body' => json_encode($body),
-            'headers' => $this->getHeaders($url, $actor, $body),
-        ]);
+        try {
+            $client = new CurlHttpClient();
+            $response = $client->request('POST', $url, [
+                'max_duration' => self::MAX_DURATION,
+                'timeout' => self::TIMEOUT,
+                'body' => $jsonBody,
+                'headers' => $this->getHeaders($url, $actor, $body),
+            ]);
 
-        if (!str_starts_with((string) $response->getStatusCode(), '2')) {
-            throw new InvalidApPostException("Post fail: $url, ".substr($response->getContent(false), 0, 1000).' '.json_encode($body));
+            $statusCode = $response->getStatusCode();
+            if (!str_starts_with((string) $statusCode, '2')) {
+                // Do NOT include the response content in the error message, this will be often a full HTML page
+                throw new InvalidApPostException("Post failed: $url, status code: $statusCode, request body: $jsonBody");
+            }
+        } catch (\Exception $e) {
+            $this->logRequestException($response, $url, 'ApHttpClient:post', $e);
```

Comment:
Apparently `logRequestException` is logging for "get failed..." .. I will either need to create a `logRequestException` or provide an additional parameter for the message "get failed" or when it's "post failed".

Reply:
or .. just remove the word "get"

Reply:
Solved.